SSOT/CSTScriptDeploymentInstructions.txt

```diff
@@ -33,3 +33,4 @@
 	12)SELECT * FROM [dbo].[OccupationalInterest]
 	13)SELECT * FROM [dbo].[OccupationInterestOption]
 	14)SELECT * FROM [dbo].[OccupationGroup]
+	15)Check the Primary Key and Foreign keys in the tables.

```

Comment:
This is not actionable: How to check the keys? What is the output that they will give you?